### PR TITLE
8304691: Remove jlink --post-process-path option

### PR DESCRIPTION
**Pull request by igraves**
Removing the hidden `jlink --post-process-path` option.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8304691](https://bugs.openjdk.org/browse/JDK-8304691): Remove jlink --post-process-path option

### Reviewers
 * [Mandy Chung](https://openjdk.org/census#mchung) (@mlchung - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13126/head:pull/13126` \
`$ git checkout pull/13126`

Update a local copy of the PR: \
`$ git checkout pull/13126` \
`$ git pull https://git.openjdk.org/jdk.git pull/13126/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13126`

View PR using the GUI difftool: \
`$ git pr show -t 13126`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13126.diff">https://git.openjdk.org/jdk/pull/13126.diff</a>

</details>

### CODE REVIEW

**Review by mlchung — COMMENTED** [review]
I expect `JlinkTask::postProcessImage` and `Jlink::postProcess` to be removed.

**Comment by mlchung** [comment]
The localized resource bundles for other languages are updated together in bulk toward the end of the release.   `plugins_xx.properties` can be kept unchanged and let them updated by the L10N team in bulk.

File: src/jdk.jlink/share/classes/jdk/tools/jlink/internal/TaskHelper.java [hunk 1]
```diff
@@ -259,13 +258,6 @@ private PluginsHelper() throws BadArgs {
                     throw newBadArgs("err.no.such.plugin", arg);
                 },
                 false, "--disable-plugin"));
-            mainOptions.add(new PluginOption(true, (task, opt, arg) -> {
-                Path path = Paths.get(arg);
-                if (!Files.exists(path) || !Files.isDirectory(path)) {
-                    throw newBadArgs("err.image.must.exist", path);
```
Comment (mlchung) [inline src/jdk.jlink/share/classes/jdk/tools/jlink/internal/TaskHelper.java:265]: "err.image.must.exist" can be removed from `jlink.properties`

**Review by mlchung — COMMENTED** [review]
`test/jdk/tools/lib/tests/JImageGenerator` references this option and needs update.

File: src/jdk.jlink/share/classes/jdk/tools/jlink/internal/JlinkTask.java [hunk 2]
```diff
@@ -371,29 +365,6 @@ public static void createImage(JlinkConfiguration config,
...
-                parseConfiguration(config);
-
-        stack.operate((ImagePluginStack stack1) -> image);
-    }
-
-    private void postProcessOnly(Path existingImage) throws Exception {
-        PluginsConfiguration config = taskHelper.getPluginsConfig(null, null);
-        ExecutableImage img = DefaultImageBuilder.getExecutableImage(existingImage);
-        if (img == null) {
-            throw taskHelper.newBadArgs("err.existing.image.invalid");
```
[the remark this answers is not in the document]
Comment (igraves) [reply src/jdk.jlink/share/classes/jdk/tools/jlink/internal/JlinkTask.java:392]: Thanks!

**Review by mlchung — APPROVED** [review]
Good to see this code deletion.  Looks good.

**Pull request closed without merging**